### PR TITLE
Add script to check SDL does not resend OnSeekMediaClockTimer if minu…

### PR DESCRIPTION
**Pull request by OVikhrov**
…tes wrong value type
@aarotar please review

### CODE REVIEW

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Wrong_Value_Type.lua [hunk 1]
```diff
@@ -0,0 +1,96 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Wrong_Value_Type.lua]: @OVikhrov, please update Requirement summary with related functional requirement also

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Wrong_Value_Type.lua [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+-- b) OnSeekMediaClockTimer notification allowed in preloaded file for default app
+-- c) App successfylly registered, consented and activated
+--
+-- 2. Performed steps:
+-- a) HMI sends OnSeekMediaClockTimer notification to SDL with wrong minutes value type (string).
+--
+-- Expected result:
+-- a) SDL does not resend OnSeekMediaClockTimer notification to mobile app.
+---------------------------------------------------------------------------------------------
+
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Wrong_Value_Type.lua]: @OVikhrov, plase avoid redundant empty lines

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Wrong_Value_Type.lua (hunk 1 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Wrong_Value_Type.lua]: Accepted

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Wrong_Value_Type.lua (hunk 2 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_minutes_Wrong_Value_Type.lua]: Accepted

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**